test/dieselpgcrate/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 authors = ["clux <sszynrae@gmail.com>"]
-name = "dieselcrate"
+name = "dieselpgcrate"
```

Comment:
this rename probably breaks the makefile `make test` should work, worth adding a target for the new test crate as well!